config/reverse_proxy/nginx.conf

```diff
@@ -0,0 +1,65 @@
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    upstream relay {
+        server sentry_relay.sentry:3000;
+    }
+
+    upstream sentry {
+        server host.docker.internal:8888;
```

Comment:
So the consensus is that we ignore Ubuntu for now?

Reply:
I didn’t ask for a consensus but the feedback I got was not to worry about Linux since according to Burak there is already a fix for host.docker.internal  for the linux docker containers https://sentry.slack.com/archives/GBTDXNA05/p1580915616030000?thread_ts=1580907540.016900&cid=GBTDXNA05.